src/Playlist.class.php

```diff
@@ -18,11 +18,6 @@
 class Playlist extends RestClient
 {
   /**
-  * Playlist holds the Playlist array
-  * @var array
-  */
-    public $playlist = null;
```

Comment:
Same thing applies here, although it's unclear what data is returned by the api since it doesn't appear to be documented.